### PR TITLE
DRY refactor constants and prepare data for all tabs

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

So far, the constants have been translations of the client-provided text copy with little modification. The constants started repeating themselves a bit, so this PR takes a crack at the constants to DRY refactor them. It also prepares all the data to be fed directly to the Screensaver (`FISH`), MeetTheFish (`FISH`), Quiz (`QUIZ_FISH`), and SeeTheFishway (`FISH_HIGHLIGHTS`) tabs. The About tab had already been taken care of.

Connects #39 

## Testing Instructions

Nothing broken.

### CODE REVIEW

File: src/app/src/constants.js [hunk 1]
```diff
@@ -1,345 +1,578 @@
 export const aboutProfiles = [
```
Comment (fungjj92) [inline src/app/src/constants.js]: Oh, let me caps and snakecase all the constants

Comment (fungjj92) [reply src/app/src/constants.js]: Completed in 1d9a373

**Comment by maurizi** [comment]
This is going to conflict with #50, which also reformatted this file somewhat. Mind if I wait to review this until after that gets merged?

**Comment by fungjj92** [comment]
Just rebased! The app is starting to look snazzy with About and fonts from the last several merges.

**Review by maurizi — APPROVED** [review]
:+1:

**Pull request merged**